util/Makefile.am

```diff
@@ -14,18 +14,13 @@ mate_menu_spec_test_SOURCES = \
 	test-menu-spec.c
 
 mate_menu_spec_test_LDADD = \
-	$(GLIB_LIBS) \
+	$(GIO_UNIX_LIBS)		\
 	../libmenu/libmate-menu.la
 
-if HAVE_PYTHON
-pyexampledir = $(pkgdatadir)/examples
-pyexample_DATA = mate-menus-ls.py
-else
-pyexampledir =
-pyexample_DATA =
-endif
+exampledir = $(pkgdatadir)/examples
+example_DATA = mate-menus-ls.py mate-menus-ls.js
```

Comment:
I don't think we need a JavaScript example. It relies on `gjs`, which is used by gnome-shell and most probably not installed in a typical system with MATE.

Reply:
Thank you for review this pr, the javascript example has been removed now.